sympy/relation/binrel.py

```diff
@@ -0,0 +1,647 @@
+
+        Examples
+        ========
+
+        >>> from sympy import Q
+        >>> from sympy.abc import x
+        >>> Q.eq(x, 1)
+        x = 1
+        >>> _.reversedsign
+        -x = -1
+        >>> Q.lt(x, 1)
+        x < 1
+        >>> _.reversedsign
+        -x > -1
+        """
+        a, b = self.arguments
+        if not (isinstance(a, BooleanAtom) or isinstance(b, BooleanAtom)):
+            return self.function.reversed(-self.lhs, -self.rhs)
+        else:
+            return self
+
+    @property
+    def canonical(self):
+        """Return a canonical form of the relational by putting a
+        number on the rhs, canonically removing a sign or else
+        ordering the args canonically. No other simplification is
+        attempted.
+
+        Examples
+        ========
+
+        >>> from sympy import Q
+        >>> from sympy.abc import x, y
+        >>> Q.lt(x,2)
+        x < 2
+        >>> _.reversed.canonical
+        x < 2
+        >>> Q.lt(-y, x).canonical
+        x > -y
+        >>> Q.gt(-y, x).canonical
+        x < -y
+        >>> Q.lt(-y, -x).canonical
+        x < y
+        """
+        args = self.arguments
+        r = self
+        if r.rhs.is_number:
+            if r.rhs.is_Number and r.lhs.is_Number and r.lhs > r.rhs:
+                r = r.reversed
+        elif r.lhs.is_number:
+            r = r.reversed
+        elif tuple(ordered(args)) != args:
+            r = r.reversed
+
+        LHS_CEMS = getattr(r.lhs, 'could_extract_minus_sign', None)
+        RHS_CEMS = getattr(r.rhs, 'could_extract_minus_sign', None)
+
+        if isinstance(r.lhs, BooleanAtom) or isinstance(r.rhs, BooleanAtom):
+            return r
+
+        # Check if first value has negative sign
+        if LHS_CEMS and LHS_CEMS():
+            return r.reversedsign
+        elif not r.rhs.is_number and RHS_CEMS and RHS_CEMS():
+            # Right hand side has a minus, but not lhs.
+            # How does the expression with reversed signs behave?
+            # This is so that expressions of the type
+            # Q.eq(x, -y) and Q.eq(-x, y)
+            # have the same canonical representation
+            expr1, _ = ordered([r.lhs, -r.rhs])
+            if expr1 != r.lhs:
+                return r.reversed.reversedsign
+
+        return r
+
+    @property
+    def binary_symbols(self):
+        return set()
+
+    def as_Relational(self):
+        return self.function.as_Relational(*self.arguments)
+
+    def simplify(self, equation=True, side="all", **kwargs):
+        """
+        Simplify *self* without evaluating to boolean value. Assumption
+        is not taken into consideration.
+
+        Parameters
+        ==========
+
+        equation : bool, optional
+            If ``True``, simplify both sides and canonical result.
+            If ``False``, each side is simplified but not canonicalized.
+            You can decide which side to be simplified by *side* argument.
```

Comment:
Would it make more sense for this to be called relation rather than equation?

Reply:
I think it will.